schema/schema.json

```diff
@@ -139,6 +139,31 @@
                 "noNewPrivileges": {
                     "id": "https://opencontainers.org/schema/bundle/process/linux/noNewPrivileges",
                     "type": "boolean"
+                },
+                "rlimits": {
+                    "id": "https://opencontainers.org/schema/bundle/linux/rlimits",
+                    "type": "array",
+                    "items": [
+                        {
```

Comment:
JSON Schema's `items` takes an object, not an array of objects.  The docs are pretty sparse here, but see [this example][1].

[1]: https://tools.ietf.org/html/draft-zyp-json-schema-04#section-7.2.3